src/reporters/lang/en.js

```diff
@@ -188,6 +188,8 @@ const messages = {
 
   execMissingCommand: 'Missing command name.',
 
+  dashDashDeprecation:
+    "Using -- to pass arguments to your scripts isn't required anymore. Doing this may cause issues in future versions.",
```

Comment:
How about: "Yarn does not require \`--\` to pass arguments to scripts. In a future version of Yarn, passing \`--\` will fail with an error." ?